src/api/auth/controller.js

```diff
@@ -136,6 +149,22 @@ const getAccessToken = async (req, res, next) => {
         next({ path: '/auth/getAccessToken', status: 400, message: err.message, extraData: err });
     }
 };
+
+// Function to fetch pull request count
+const getPullRequestCount = async (githubId, accessToken) => {
+    try {
+        const response = await axios.get(`https://api.github.com/search/pulls?q=author:${githubId}+is:public`, {
```

Comment:
why fetching user's all prs, we don't want his all pr count, we want pr counts of our projects, so you can remove this thing